### PR TITLE
Adds navbar to scheduling page

### PR DESCRIPTION
**Pull request by meagancheese**
Also adds this page's link to all other navbars.
Screenshot 2020-06-25 at 5 54 25 PM

### CODE REVIEW

File: src/main/webapp/search-interview.html [hunk 1]
```diff
@@ -17,7 +17,27 @@
...
+        <span class="navbar-brand mb-0 h1">InterviewMe</span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" 
+          data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" 
+          aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <a class="nav-item nav-link" href="index.html">Home</a>
+            <!--TODO: Put in links to other pages. Put on all pages-->
```
Comment (apluscs) [inline src/main/webapp/search-interview.html]: We can probably remove these TODOs now.

**Review by apluscs — CHANGES_REQUESTED** [review]
QQ is this the last part of our navbar integration?

File: src/main/webapp/search-interview.html (hunk 1 above)
Comment (meagancheese) [reply src/main/webapp/search-interview.html]: Done

**Comment by meagancheese** [comment]
Comments addressed, PTAL.

**Review by apluscs — APPROVED** [review]

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**